source/retryable-reads/tests/gridfs-downloadByName-serverErrors.yml

```diff
@@ -0,0 +1,149 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "files"
+
+data:
+    files:
+        -
+            _id: { "$oid" : "000000000000000000000001" }
+            length: 1
+            chunkSize: 4
+            uploadDate: { "$date" : "1970-01-01T00:00:00.000Z" }
+            md5: "47ed733b8d10be225eceba344d533586"
+            filename: "abc"
+            contentType: "application/octet-stream"
+            aliases: []
+            metadata: {}
+    chunks:
+        - { _id : { "$oid" : "000000000000000000000001" }, files_id : { "$oid" : "000000000000000000000001" }, n : 0, data : { $hex : "11" } }
+
+tests:
+    -
+        description: "Download_by_name succeeds after InterruptedAtShutdown"
```

Comment:
`DownloadByName`? Likewise for other YAML files where you used underscore naming instead of camel case.